7Cav-Alive.Altis/scripts/missionEnd/missionEnd_StrongPoint.sqf

```diff
@@ -0,0 +1,45 @@
+/* This script will:
+ *
+ * Define some parameters, log which box was activated, Initialize some variables, remove the initial addaction, 
+ * and then perform some user feedback. The user feedback is achieved by animating the box, completing a task,
+ * then changing the orbat markers on the map.
+ *
+ * After the user feedback, we check if all the boxes were activated. If they were, we end the mission. If they
+ * weren't, we do nothing.
+ *
+ */
+
+//Addaction return values for the boxes, so we can get the global variable
+params ["_target", "_caller", "_actionId", "_arguments"];
+
+//Set the local variable equal to the global variable
+private _boxNumber = missionNameSpace getVariable "endMissionNumber";
+
+//log which box was activated
+diag_log formatText["%1 has been activated!", _target];

```

Comment:
We don't have a debug log system and we don't want to print random logs, so remove or comment out this line.